packages/server/src/schema/metadata/resolvers.ts

```diff
@@ -84,5 +84,10 @@ export const resolvers: Queries<'metadata'> = {
         },
       }
     },
+    version() {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires
+      const { version } = require('../../../package')
+      return version as string
```

Comment:
```suggestion
  return process.env.npm_package_version as string
```
I've liked your previous implementation, it is cleaner... ;)

Reply:
me too! 
Unfortunately `process.env.npm_package_version ` returns the version of the `package.json` in the root here. (which is `0.0.0`).
Do you have an idea how to change that?
Otherwise the version above works.

I added a testcase to make sure version it not `0.0.0`.

Reply:
Strangely enough, I've tested manually and it usually gives back the right version, but just one time I got the 0.0.0 as you mentioned, I'm not sure what we have done that caused this strange behavior. I can do the change depoy and we see what comes. In case of wrong version, I'll make a fix.